themes/stanford_basic/theme-settings.php

```diff
@@ -37,6 +37,22 @@ function stanford_basic_form_system_theme_settings_alter(array &$form, FormState
     '#default_value' => theme_get_setting('brand_bar_variant'),
   ];
 
+  // Global footer support.
+  $form['options_settings']['stanford_basic_global_footer'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Global Footer Settings'),
+  ];
+
+  $form['options_settings']['stanford_basic_global_footer']['global_footer_variant'] = [
+    '#type' => 'select',
+    '#title' => t('Global Footer Variant'),
+    '#options' => [
+      'default' => '- Default -',
+      'dark' => t('Dark'),
+    ],
```

Comment:
instead of a `default` option, use `#empty_option`: https://api.drupal.org/api/drupal/developer%21topics%21forms_api_reference.html/7.x#empty_option
Make sure it is is in the `t()` function too.

Reply:
Neat!